lib/wasi.js

```diff
@@ -48,28 +48,22 @@ class WASI {
     validateObject(options, 'options');
 
     let _WASI;
-    if (options.version !== undefined) {
-      validateString(options.version, 'options.version');
-      switch (options.version) {
-        case 'unstable':
-          ({ WASI: _WASI } = internalBinding('wasi'));
-          this[kBindingName] = 'wasi_unstable';
-          break;
-        // When adding support for additional wasi versions add case here
-        case 'preview1':
-          ({ WASI: _WASI } = internalBinding('wasi'));
-          this[kBindingName] = 'wasi_snapshot_preview1';
-          break;
-        // When adding support for additional wasi versions add case here
-        default:
-          throw new ERR_INVALID_ARG_VALUE('options.version',
-                                          options.version,
-                                          'unsupported WASI version');
-      }
-    } else {
-      // TODO(mdawson): Remove this in a SemVer major PR before Node.js 20
-      ({ WASI: _WASI } = internalBinding('wasi'));
-      this[kBindingName] = 'wasi_snapshot_preview1';
+    validateString(options.version, 'options.version');
+    switch (options.version) {
+      case 'unstable':
+        ({ WASI: _WASI } = internalBinding('wasi'));
+        this[kBindingName] = 'wasi_unstable';
+        break;
+      // When adding support for additional wasi versions add case here
```

Comment:
I think this comment can be removed (it's duplicated a few lines down).

Reply:
removed